lib/Monitor/SRS/Local.pm

```diff
@@ -87,10 +87,10 @@ sub is_run_completed {
 
     my $run_folder = Monitor::RunFolder->new( runfolder_path => $run_path, _schema => $self->schema );
```

Comment:
yes

On 19/08/15 19:33, dkj wrote:
>
> In lib/Monitor/SRS/Local.pm 
> <https://github.com/wtsi-npg/npg_tracking/pull/288#discussion_r37450371>:
>
> > @@ -87,10 +87,10 @@ sub is_run_completed {
> >
> >      my $run_folder = Monitor::RunFolder->new( runfolder_path => $run_path, _schema => $self->schema );
>
> Can we get rid of this?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/wtsi-npg/npg_tracking/pull/288/files#r37450371>.
>




-- 
 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.